packages/react/dropdown-menu/src/DropdownMenu.stories.tsx

```diff
@@ -473,7 +473,7 @@ export const Chromatic = () => {
           </DropdownMenuItem>
           <DropdownMenuSeparator className={separatorClass} />
           <DropdownMenu open>
-            <DropdownMenuTrigger className={subTriggerClass}>Submenu →</DropdownMenuTrigger>
+            <DropdownMenuTriggerItem className={subTriggerClass}>Submenu →</DropdownMenuTriggerItem>
```

Comment:
A bunch of these were not updated, I just realised because Chromatic flagged it.

Reply:
Ah nice, I just sat down to look at it but you already did 👍